app/components/debug-grids.hbs

```diff
@@ -0,0 +1,101 @@
+          <li>
+            <p data-test-debug-social-id={{social}}><span
+                class="debug--bold"
+              >{{social}}:</span>
+              {{socialId}}</p>
+          </li>
+        {{/each-in}}
+      </ul>
+    </div>
+    <div data-test-debug-user class="debug__user debug__grid-item">
+      <ul>
+        {{#each-in this.debugUserData as |key value|}}
+          <li>
+            <p data-test-debug-user-data={{key}}><span class="debug--bold">
+                {{key}}:</span>
+              {{value}}</p>
+          </li>
+        {{/each-in}}
+        <li class="debug__user__roles-container">
+          <p class="debug--bold">Roles:</p>
+          <ul>
+            {{#each-in this.debugUserRolesData as |role value|}}
+              <li><p data-test-debug-role={{role}}><span
+                    class="debug--semibold"
+                  >{{role}}:</span>
+                  {{value}}</p></li>
+            {{/each-in}}
+          </ul>
+        </li>
+      </ul>
+    </div>
+    <div data-test-debug-features class="debug__features debug__grid-item">
+      <div class="debug__features__feature-flag">
+        <h3 class="debug--bold">Features Flags:</h3>
+        <ul data-test-debug-feature-flags>
+          {{#each this.debugFeaturesData.featureFlags as |featureFlag|}}
+            <li><p>{{featureFlag}}</p></li>
+          {{/each}}
+        </ul>
+      </div>
+      <div data-test-debug class="debug__features__superuser">
+        <div class="debug__features__superuser__indicator-container">
+          <h3 class="debug--bold">Super User</h3>
+          <div
+            data-test-debug-superuser-indicator
+            class="debug__features__superuser__indicator
+              {{if
+                this.debugFeaturesData.isSuperUser
+                'debug__features__superuser__indicator--active'
+              }}"
+          ></div>
+        </div>
+        <p
+          data-test-debug-superuser-message
+          class="debug__features__superuser__message"
+        >{{#if this.debugFeaturesData.isSuperUser}}You're a super user.
+            Remember, with great power comes great responsibilities!{{else}}You're
+            not a super user.{{/if}}</p>
+      </div>
+      {{#if this.debugFeaturesData.isSuperUser}}
+        <div class="debug__features__superuser-privileges">
+          <h3 class="debug--semibold">Apply privileges: </h3>
+          <div class="privileges-toggle__container">
+            <label
+              data-test-debug-apply-privileges-button
+              class="privileges-toggle"
+            >
+              <input type="checkbox" class="privileges-toggle__checkbox" />
+              <span class="privileges-toggle__slider"></span>
+            </label>
+          </div>
+        </div>
+      {{/if}}
+    </div>
+  </div>
+{{else}}
+  <div data-test-debug-error-message class="debug__error-message">
+    <p>Something went wrong... </p>
+    <p>Please try again.</p>
```

Comment:
Error should be on point and guide user what to do next(if possible there should be navigation to the home)

Reply:
`Please Sign In to load debug data` is this message fine?